src/plugins/controls/public/options_list/components/options_list_control.tsx

```diff
@@ -87,6 +110,8 @@ export const OptionsListControl = ({
   );
 
   const { hasSelections, selectionDisplayNode, validSelectionsCount } = useMemo(() => {
+    const delimiter = fieldSpec?.type === 'date' ? '; ' : OptionsListStrings.control.getSeparator();
```

Comment:
Could we make `OptionsListStrings.control.getSeparator()` take an argument of `type` instead, like you did for the sorting options?

Reply:
Addressed in [a38b39e](https://github.com/elastic/kibana/pull/164362/commits/a38b39e8ddc21d6d0f67d0960047627f7e780cda)